packages/p2p/src/components/my-profile/block-user/block-user-table/block-user-table.jsx

```diff
@@ -33,7 +45,7 @@ const BlockUserTable = () => {
                         <InfiniteDataList
                             data_list_className='block-user__data-list'
                             has_more_items_to_load={false}
-                            items={my_profile_store.blocked_advertisers_list}
+                            items={my_profile_store.rendered_blocked_advertisers_list}
                             keyMapperFn={item => item.id}
                             loadMoreRowsFn={() => {}}
```

Comment:
I did not add a `loadMoreRowsFn` function since the initial WS for `p2p_advertiser_relations` already fetches all the blocked advertisers and has no `offset` parameter or pagination, thus the search box also operates on filtering and does not re-fetches the list of blocked advertisers when user is searching something